app/src/main/res/layout/activity_market_picker.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This screen is starting to get tall enough that it might get clipped on smaller devices. Should we have a `ScrollView` as a parent, perhaps?

The Continue-button could be floating on top by means of a `FrameLayout`.